.classpath

```diff
@@ -2,7 +2,7 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.USER_LIBRARY/JNA"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="lib" path="C:/Users/Owner/Desktop/enigma-dev/plugins/shared/jna.jar"/>
+	<classpathentry kind="lib" path="C:/Users/Owner/Documents/projects/enigma-dev/plugins/shared/jna.jar"/>

```

Comment:
This doesn't strike me as a fix for anything.

Reply:
That jar is always external, and it has to be somewhere. I'm the one who builds it, so obviously I set it to where I usually have it, and this is where I decided to start keeping my projects.

We can look at making it relative to the project, which it might originally have been.

Reply:
Ok 35e017bf48fd8020bb43a1c567dfbfa474d36373 I made it relative.